src/main/resources/org/apache/fluo/resources/java-checkstyle.xml

```diff
@@ -30,6 +28,8 @@
   </module>
 
   <module name="TreeWalker">
+    <module name="SuppressWithNearbyCommentFilter"/>
```

Comment:
I thought that's basically what my log message said, but I can reword it if it's not clear.